README.md

```diff
@@ -20,3 +20,7 @@ The user of the app will be able to:
 
 ## Installation
 coming soon
+
+
+
+[![Coverage Status](https://coveralls.io/repos/github/ljte/lab-project/badge.svg?branch=working_branch)](https://coveralls.io/github/ljte/lab-project?branch=working_branch)
```

Comment:
Please, add the [Build Status] section as well.

Reply:
done